app/helpers/documents_helper.rb

```diff
@@ -9,12 +9,15 @@ def format_document_date(date)
   end
 
   def attached_file_formatter(document)
-    if document.completed?
+    case document.state
+    when 'Completed'
       content_tag(:a, class: 'attached_file', id: "file_#{document.id}", href: document_path(document), target: :blank, title: 'Download File', 'data-id' => document.id) do
         content_tag(:span, '', class: 'glyphicon glyphicon-file')
       end
-    else
+    when 'Pending'
```

Comment:
If we go through with removing the "Pending" status from Documents, this should be updated to "Processing".